packages/hello-world/package.json

```diff
@@ -1,36 +1,21 @@
 {
     "name": "hello-world",
-    "version": "1.5.0-dev",
+    "version": "2.0.0-dev.2",
     "license": "See license in LICENSE",
     "private": true,
-    "dependencies": {
-        "aws-serverless-express": "^3.3.6",
-        "express": "^4.17.1"
-    },
     "devDependencies": {
-        "@babel/cli": "^7.14.3",
-        "@babel/core": "^7.14.3",
-        "@babel/runtime": "^7.14.0",
-        "babel-eslint": "^10.1.0",
-        "eslint": "7.31.0",
-        "eslint-config-prettier": "4.0.0",
-        "eslint-import-resolver-webpack": "0.10.0",
-        "eslint-plugin-header": "^3.1.1",
-        "eslint-plugin-import": "2.23.4",
-        "eslint-plugin-jsx-a11y": "6.4.1",
-        "eslint-plugin-prettier": "3.0.1",
-        "eslint-plugin-react": "7.24.0",
-        "prettier": "1.18.2",
-        "pwa-kit-react-sdk": "^1.5.0-dev"
+        "pwa-kit-build": "^2.0.0-dev.2",
+        "pwa-kit-runtime": "^2.0.0-dev.2"
```

Comment:
I'm not worried about this, personally. I think we need stronger motivations that "what if people are confused". Compare these two questions we could ask about splitting packages:

1. Would it be clear to others which packages to download for their projects?
2. What capabilities do we provide by splitting the packages in this way?

It's very hard to answer (1) and very easy to answer (2). For (1), we really don't know how clear it would be. But we solve this problem around "clarity" already by providing project templates. To me, this doesn't seem to be a thing worth worrying about.

But (2) is absolutely crucial to do: We want to support different types of applications. And we want to use our same runtime, with proxies, etc. So, if we want people to be able to write a GQL server, a REST API, an LWR application and use our runtime – well, we _have_ to split the runtime from our React SDK. Otherwise people can only build React apps!

The answer to (2), then, is "we get the ability to run different types of applications if we split these packages". In my opinion, that is a _very_ strong reason to split code!

Thoughts?

Reply:
Yes, I see now.. the advantage offered by 2 definitely outweighs the cost/disadvantage.